logo/logo_base.py

```diff
@@ -17,4 +25,31 @@
 comp = pyna.Composition(rc.get_nuclei())
 comp.set_solar_like()
 
-rc.plot(rho=2.e6, T=3.e7, comp=comp, outfile="cno_flow.png")
+fig = rc.plot(rho=2.e6, T=3.e7, comp=comp,
+              node_size=1300, node_font_size=18, node_color="#444444")
+
+# ax[0] is the plot, ax[1] is the colorbar
+ax = fig.get_axes()
+ax[0].set_xlim(5.5, 8.5)
+ax[0].set_ylim(5.5, 8.2)
+
+print(ax)
+
+ax[0].set_axis_off()
+ax[1].remove()
+
+fig.text(0.42, 0.1, "pynucastro", horizontalalignment="center",
+         color="#444444",
+         fontsize="40", fontstyle="italic", fontfamily="Fira Sans")
+
+ax[0].set_aspect("equal", adjustable="datalim")
+
+fig.set_size_inches(4.5, 4.0)
+
+
+ax[0].margins(0.0)
+fig.tight_layout()
+
+fig.savefig("logo.pdf", bbox_inches="tight")
+fig.savefig("logo.svg", bbox_inches="tight")
```

Comment:
The PDF and SVG have some extra padding as compared to the PNG. Does `pad_inches=0` work for those writers too?

Reply:
indeed, it seems to help